terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java

```diff
@@ -16,7 +16,11 @@
 package org.terasoluna.tourreservation.app.managereservation;
```

Comment:
- [ ] It seems that there are other places where SimpleDateFormat is used besides this class. Please correct them.

Reply:
The following source is fixed besides ReservationReportPdfStamperView.java

> application-messages_en.properties
> application-messages_ja.properties
> ManageReservationHelper.java
> ReservationReportPdfStamperViewTest.java